src/app/core/services/phase.service.ts

```diff
@@ -118,19 +118,6 @@ export class PhaseService {
     this.githubService.storePhaseDetails(this.phaseRepoOwners[this.currentPhase], this.repoName);
   }
 
-  /**
-   * Launches the SessionFixConfirmation Dialog.
-   * @return Observable<boolean> - Representing user's permission grant.
-   */
-  openSessionFixConfirmation(): Observable<boolean> {
-    const dialogRef = this.phaseFixConfirmationDialog.open(SessionFixConfirmationComponent, {
```

Comment:
Let's remove the `phaseFixConfirmationDialog` from the dependencies of phase service, since it's no longer used in this service.

Reply:
Alright! I forgot to remove it... my bad ><